### PR TITLE
chore: bump test image to latest with kpt beta 13

### PR DESCRIPTION
**Pull request by bharathkkb**

### CODE REVIEW

**Comment by bharathkkb** [comment]
Looks like its failing in CI due to timeout at destroy. Opened https://github.com/GoogleContainerTools/kpt/issues/2687

```
2022/01/28 20:17:46 Running stage teardown
TestSimpleBucket 2022-01-28T20:17:46Z command.go:100: Running command kpt with args [live destroy]
TestSimpleBucket 2022-01-28T20:17:46Z command.go:179: storagebucket.storage.cnrm.cloud.google.com/ci-blueprints-10795-test-bucket deleted
TestSimpleBucket 2022-01-28T20:17:46Z command.go:179: 1 resource(s) deleted, 0 skipped, 0 failed to delete
TestSimpleBucket 2022-01-28T20:17:46Z command.go:179: storagebucket.storage.cnrm.cloud.google.com/ci-blueprints-10795-test-bucket reconcile pending
{"component":"entrypoint","file":"prow/entrypoint/run.go:165","func":"k8s.io/test-infra/prow/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 1h0m0s timeout","severity":"error","time":"2022-01-28T20:59:45Z"}
{"component":"entrypoint","file":"prow/entrypoint/run.go:255","func":"k8s.io/test-infra/prow/entrypoint.gracefullyTerminate","level":"error","msg":"Process did not exit before 15s grace period","severity":"error","time":"2022-01-28T21:00:00Z"}
```

File: .prow.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ presubmits:
       timeout: 30m
     spec:
       containers:
-        - image: gcr.io/cloud-foundation-cicd/cft/developer-tools-krm:1.1.3
+        - image: gcr.io/cloud-foundation-cicd/cft/developer-tools-krm:1.3.6
```
Comment (morgante) [inline .prow.yaml:26]: We probably shouldn't pin to a patch version unless/until we have automation to update this automatically.

**Review by morgante — APPROVED** [review]

**Comment by bharathkkb** [comment]
bumped version in https://github.com/GoogleCloudPlatform/blueprints/pull/146

**Pull request closed without merging**